### PR TITLE
Added keybinds hint shortcut

### PR DESCRIPTION
**Pull request by XBEAST1**
Added keybinds hint shortcut

### CODE REVIEW

**Comment by kRHYME7** [comment]
@XBEAST1   This is approved, but I will try to open a Poll in the discord server first to further know which bind is intuitive for new HyDE users.

File: Configs/.config/hypr/keybindings.conf [hunk 1]
```diff
@@ -79,6 +79,7 @@ bind = $mainMod+Shift, A, exec, pkill -x rofi || $scrPath/rofiselect.sh # launch
 bind = $mainMod+Shift, W, exec, pkill -x rofi || $scrPath/swwwallselect.sh # launch wallpaper select menu
 bind = $mainMod, V, exec, pkill -x rofi || $scrPath/cliphist.sh c # launch clipboard
 bind = $mainMod, K, exec, $scrPath/keyboardswitch.sh # switch keyboard layout
+bind = $mainMod, H, exec, pkill -x rofi || $scrPath/keybinds_hint.sh c # launch keybinds hint
```
Comment (kRHYME7) [inline Configs/.config/hypr/keybindings.conf:82]: How about changing H into slash? 

`
bind = $mainMod, slash, exec, pkill -x rofi || $scrPath/keybinds_hint.sh c # launch keybinds hint
`

* slash on keyboard had the same position with ` question ` "?"  which  which might be more intuitive. 
* ` H `  as per vim keybindings will conflict on this  too
* mostly linux uses `/` to search/query/help

**Review by kRHYME7 — CHANGES_REQUESTED** [review]

File: Configs/.config/hypr/keybindings.conf (hunk 1 above)
Comment (XBEAST1) [reply Configs/.config/hypr/keybindings.conf:82]: these keys aren't working when i try to bind . or / or ; do you know why? i searched everywhere for this i also wanted to bind ; with my emojis but it wasn't working as well

Comment (XBEAST1) [reply Configs/.config/hypr/keybindings.conf]: Done

**Review by XBEAST1 — COMMENTED** [review]
in readme what do you think which will be better?

<kbd>Super</kbd> + <kbd>slash</kbd> | Launch keybinds hint |

or

<kbd>Super</kbd> + <kbd>/</kbd> | Launch keybinds hint |

**Comment by kRHYME7** [comment]
the keybinds hint detects it as slash and we can't really use bare "/" in the .conf file. 
image
So we will use slash instead.

**Review by kRHYME7 — APPROVED** [review]
Thank you!

File: README.md [hunk 2]
```diff
@@ -176,6 +176,7 @@ For more information, visit [prasanthrangan/hyde-themes](https://github.com/pras
 | <kbd>Super</kbd> + <kbd>Del</kbd> | Kill Hyprland session |
 | <kbd>Super</kbd> + <kbd>W</kbd> | Toggle the window between focus and float |
 | <kbd>Super</kbd> + <kbd>G</kbd> | Toggle the window between focus and group |
+| <kbd>Super</kbd> + <kbd>H</kbd> | Launch keybinds hint |
```
[the remark this answers is not in the document]
Comment (dieBakterie) [reply README.md]: Why is it $scrPath/keybinds_hint.sh **c**? Where does the **c** comes from?

**Comment by XBEAST1** [comment]
> the keybinds hint detects it as slash and we can't really use bare "/" in the .conf file. image So fwe will use slash instead.



> the keybinds hint detects it as slash and we can't really use bare "/" in the .conf file. image So we will use slash instead.

got it 👍

File: Configs/.config/hypr/keybindings.conf [hunk 3]
```diff
@@ -79,6 +79,7 @@ bind = $mainMod+Shift, A, exec, pkill -x rofi || $scrPath/rofiselect.sh # launch
 bind = $mainMod+Shift, W, exec, pkill -x rofi || $scrPath/swwwallselect.sh # launch wallpaper select menu
 bind = $mainMod, V, exec, pkill -x rofi || $scrPath/cliphist.sh c # launch clipboard
 bind = $mainMod, K, exec, $scrPath/keyboardswitch.sh # switch keyboard layout
+bind = $mainMod, slash, exec, pkill -x rofi || $scrPath/keybinds_hint.sh c # launch keybinds hint
```
[the remark this answers is not in the document]
Comment (XBEAST1) [reply Configs/.config/hypr/keybindings.conf:82]: > @XBEAST1 Can you check if
> 
> ```
> bindd = SUPER, slash,Show keybind hints, exec, keybinds_hint.sh # Show keybind hints
> ```
> 
> works too? I think it is better to start using this convention now. But can you check if this is posible in stable hyprland now? I am using the git

yes i've checked this working flawlessly as well

**Pull request merged**